### ISSUE TITLE
Cleaning up Joomla before upgrading to J4

### ISSUE THREAD

**Issue by rgtr** [joomla/joomla-cms#35622]
### Steps to reproduce the issue
Get an auld J1.5 site now in J3.10.2

### Expected result
All works

### Actual result
Problems with beez3, simplepie, hathor, and p3p

### System information (as much as possible)
same on PHP/8.0.6 under Win7.1 and Linux on fasthost php 7.4.16

### Additional comments
I have tested all of my sites and report is published here:
http://tdocplus.co.uk/0a_Empty400/migrating-to-j4/early-joomla-sites-manual-cleaning

This has been an exhaustive testing regime which I did on 3.10.1 and 4.0.2 and have now repeated on 3.10.2 and 4.0.3

Many people will simply not be able to do manual cleaning.

It appears that many of the auld bugs have been fixed. Hope this helps the team.

**Comment by brianteeman** [comment]
As I told you privately by email the problem was that **you** must have tried to uninstall those templates/plugins in the past and because they are core templates/plugins the files came back which is why they were able to be _discovered_. This is nothing to do with Joomla 4 as they could have been _discovered_ at any time in the life of joomla 3 after you had failed to remove them. The files would have been removed on update to joomla 4 anyway
 
As I also told you privately by email there was no need to manually delete the files you listed as they are removed in the upgrade process. As can be seen https://github.com/joomla/joomla-cms/blob/f575a49103e29e94f1d62dfa23b4ae53e680879b/administrator/components/com_admin/script.php#L638

You did also mention another issue in your emails regarding the database structure and I offered to check that if you sent me your database but you chose not to.

In summary I really do not know what you expect to be done.

**Comment by Quy** [comment]
Can this be closed?

**Comment by rgtr** [comment]
Yes of course.
The cleaning up process remains .... 
Richard
